### PR TITLE
Remove unused variables from generated stub files

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/2382
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina-tests/tests/resources/generated-sources/tool_test_data_type_1/helloWorldWithDependency_pb.bal [hunk 1]
```diff
@@ -33,8 +33,8 @@ public isolated client class helloWorldWithDependencyClient {
         } else {
             message = req;
         }
-        var payload = check self.grpcClient->executeSimpleRPC("helloWorldWithDependency/hello", message, headers);
-        [anydata, map<string|string[]>] [result, respHeaders] = payload;
+        var _ = check self.grpcClient->executeSimpleRPC("helloWorldWithDependency/hello", message, headers);
+        [anydata, map<string|string[]>] [_, respHeaders] = payload;
         return {content: <HelloResponse>result, headers: respHeaders};
```
Comment (BuddhiWathsala) [inline ballerina-tests/tests/resources/generated-sources/tool_test_data_type_1/helloWorldWithDependency_pb.bal:38]: I couldn’t find where this `result` come from.

File: ballerina-tests/tests/resources/generated-sources/tool_test_data_type_11/oneof_field_service_pb.bal [hunk 2]
```diff
@@ -33,8 +33,8 @@ public isolated client class OneofFieldServiceClient {
         } else {
             message = req;
         }
-        var payload = check self.grpcClient->executeSimpleRPC("grpcservices.OneofFieldService/hello", message, headers);
-        [anydata, map<string|string[]>] [result, respHeaders] = payload;
+        var _ = check self.grpcClient->executeSimpleRPC("grpcservices.OneofFieldService/hello", message, headers);
+        [anydata, map<string|string[]>] [_, respHeaders] = payload;
         return {content: <Response1>result, headers: respHeaders};
```
Comment (BuddhiWathsala) [inline ballerina-tests/tests/resources/generated-sources/tool_test_data_type_11/oneof_field_service_pb.bal:38]: Same unknown `result` here.

File: ballerina-tests/tests/resources/generated-sources/tool_test_data_type_20/multiple_wrapper_types_pb.bal [hunk 3]
```diff
@@ -147,88 +147,88 @@ public isolated client class HelloWorld3Client {
...
+        var _ = check self.grpcClient->executeSimpleRPC("grpcservices.HelloWorld3/testIntArrayOutput", message, headers);
+        [anydata, map<string|string[]>] [_, respHeaders] = payload;
         return {content: <TestInt>result, headers: respHeaders};
     }
 
     isolated remote function testStringArrayOutput() returns TestString|grpc:Error {
         empty:Empty message = {};
         map<string|string[]> headers = {};
-        var payload = check self.grpcClient->executeSimpleRPC("grpcservices.HelloWorld3/testStringArrayOutput", message, headers);
+        var _ = check self.grpcClient->executeSimpleRPC("grpcservices.HelloWorld3/testStringArrayOutput", message, headers);
```
Comment (dilanSachi) [inline ballerina-tests/tests/resources/generated-sources/tool_test_data_type_20/multiple_wrapper_types_pb.bal:174]: is it possible to remove `payload` from here and refer it in the next line

File: ballerina-tests/tests/resources/generated-sources/tool_test_data_type_20/multiple_wrapper_types_pb.bal [hunk 4]
```diff
@@ -49,7 +49,7 @@ public isolated client class HelloWorld3Client {
         } else {
             message = req;
         }
-        var payload = check self.grpcClient->executeSimpleRPC("grpcservices.HelloWorld3/testStringArrayInput", message, headers);
+        var _ = check self.grpcClient->executeSimpleRPC("grpcservices.HelloWorld3/testStringArrayInput", message, headers);
         [anydata, map<string|string[]>] [result, _] = payload;
```
Comment (BuddhiWathsala) [inline ballerina-tests/tests/resources/generated-sources/tool_test_data_type_20/multiple_wrapper_types_pb.bal:53]: The variable `payload` is also missing here, I wonder how it compiled?

File: ballerina-tests/tests/resources/generated-sources/tool_test_data_type_1/helloWorldWithDependency_pb.bal (hunk 1 above)
Comment (MadhukaHarith92) [reply ballerina-tests/tests/resources/generated-sources/tool_test_data_type_1/helloWorldWithDependency_pb.bal:38]: Will check

**Comment by MadhukaHarith92** [comment]
Closing since need to do more testing.

**Pull request closed without merging**